open_academy/models/res_parner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
```

Comment:
The file name is almost fine, but the `t` is missing from res_par**t**ner.py Please rename it and all its references.